internal/jimm/ssh/ssh.go

```diff
@@ -112,3 +119,42 @@ func (s *sshManager) ControllerInfoFromModelUUID(ctx context.Context, modelUUID
 		JWT:       string(jwt),
 	}, nil
 }
+
+// DialControllerSSHServer determines which controller holds the desired
+// model and dials the controller returning an SSH connection to the controller.
+// We intentionally don't accept user input for the port to disallow the
+// user from probing for open ports on the Juju controller.
+func (s *sshManager) DialControllerSSHServer(ctx context.Context, modelUUID string, user *openfga.User) (*gossh.Client, error) {
+	connInfo, err := s.controllerInfoFromModelUUID(ctx, modelUUID, user)
+	if err != nil {
+		return nil, err
+	}
+	// TODO: Dial the controller and request it's SSH port
+	// here or save it when we add a controller to JIMM.
+	destPort := jujuSSHDefaultPort
+	var client *gossh.Client
+	var errs []error
+
+	for _, addr := range connInfo.Addresses {
+		dest := net.JoinHostPort(addr, fmt.Sprint(destPort))
+		client, err = gossh.Dial("tcp", dest, &gossh.ClientConfig{
+			User: "jimm",
+			//nolint:gosec // this will be removed once we handle hostkeys
+			HostKeyCallback: gossh.InsecureIgnoreHostKey(),
```

Comment:
TODO: don't ignore host key eventually? + Ticket? If we don't put todo in be harder to find and maybe put it on a line separate from the linting line, otherwise might get missed a bit

Reply:
I'll just make a card.